### DISCUSSION TITLE
LOTW certificate information display following renewal of a cert

### DISCUSSION THREAD

**Discussion by timkirby** [magicbug/Cloudlog#1867] [Ideas / Feature Requests]
Following the expiry of my GW4VXE LOTW certificate, the ARRL sent me a new one. I've duly installed the P12 file into Cloudlog.

It looks like the QSO End Date field has not been refreshed following the installation of the new cert. In my case it is still displaying 19/12/22.

![LOTW certs](https://user-images.githubusercontent.com/25195355/209351705-27c51f53-258e-44cc-a5f0-4cb7e0f583d4.png)

You can see from the information from the certificate itself that the QSO end date for the new certificate is 21/12/25.
![TQSL cert](https://user-images.githubusercontent.com/25195355/209351925-4a662179-75ba-4eca-b00f-0c4ef9e5ed71.png)

I assume this wouldn't stop Cloudlog uploading LOTW - but it looks like if you reload a certificate, it does not refresh the QSO End Date field.Date Created and Date Expired look fine.

Thank you for all your help and I'm sorry I let me certificate lapse :-)

**Comment by phl0** [comment]
I gues you did not delete the old cert before you uploaded the new one?

**Comment by phl0** [comment]
And also the date under "Status" is the one of the old cert?

**Comment by timkirby** [comment]
Yes it is and you’re right I didn’t delete the old certificate first. I can see that is a cleaner approach - but of course, in my mind, I was replacing the same cert…

I’m happy to put it down to user error, delete the cert and add it in again, if that seems better?

**Reply by phl0** [reply]
Well I think overwriting should also work as most of us would do it. Need to check the routines here. Can you confirm that everything is clean after removing the cert and uploading the new one?

**Comment by timkirby** [comment]
Yes, deleting and adding the cert in again works fine. It said ’not synched’ - so I ran a manual sync and that completed as it should.
